Formula/a/apkleaks.rb

```diff
@@ -0,0 +1,43 @@
+class Apkleaks < Formula
+  include Language::Python::Virtualenv
+
+  desc "Scanning APK file for URIs, endpoints & secrets"
+  homepage "https://github.com/dwisiswant0/apkleaks"
+  url "https://github.com/dwisiswant0/apkleaks/archive/refs/tags/v2.6.1.tar.gz"
+  sha256 "c22557f527ee49c04947c1bed1b5ee50857ee68b0f4f3b99a5e4b18dffe30d16"
+  license "Apache-2.0"
+
+  depends_on "jadx"
+  depends_on "python-click"
+  depends_on "python-lxml"
+  depends_on "python-setuptools"
```

Comment:
Should these be listed in the `pypi_formula_mappings.json` file?

Reply:
ideally, yes, but since it does not have pypi artifact, I did not add one, I can populate one though